members/CusJa11440fb1646939689.yaml

```diff
@@ -54,6 +54,12 @@ contact_form:
           options:
             max_length: 900
           required: true
+    - javascript:
+      - name: char replace
+        values: ["$MESSAGE"]
+        selectors: ["#ctl00_ContentPlaceHolderMain_txtMessage"]
+        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(-)/g,'');}"]
```

Comment:
since there's a max length on the message, we should use the same pattern we used here https://github.com/NGPVAN/contact-congress/pull/1376/files where we used the value already in the textbox instead of what is in `$MESSAGE`

Reply:
fixed it!